01week/rockPaperScissors.js

```diff
@@ -8,10 +8,28 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+function rockPaperScissors(arg1, arg2) {
+  
+  const hand1 = arg1.trim().toLowerCase();
+  const hand2 = arg2.trim().toLowerCase();
+  //console.log(hand1,hand2)
 
-  // Write code here
+  if (hand1===hand2){
+    return "It's a tie";
+  }else if
+  (hand1==='rock' && hand2==='scissors'||
+  hand1==='scissors' && hand2==='paper'||
+hand1==='paper' && hand2==='rock'){
```

Comment:
Be sure to provide adequate spacing in your code.  Start developing the habit of providing line tabs.  This helps with readability for you and other developers that need to read your code.

Reply:
Fixed.